lib/scala/project-manager/src/main/scala/org/enso/projectmanager/infrastructure/languageserver/HeartbeatSession.scala

```diff
@@ -144,6 +161,22 @@ class HeartbeatSession(
     context.become(socketClosureStage(closureTimeout))
   }
 
+  private def logError(throwable: => Throwable, message: String): Unit = {
+    if (quietErrors) {
+      log.debug(s"$message ($throwable)")
+    } else {
+      log.error(throwable, message)
+    }
+  }
+
+  private def logError(message: String): Unit = {
+    if (quietErrors) {
+      log.debug(message)
+    } else {
+      log.error(message)
+    }
+  }
+
```

Comment:
That's because these errors are 'expected'.

We start sending the initial heartbeats right away after starting the process. The first few heartbeats will 99% of the time fail as the LS has not finished booting when they are sent. So this error is not a 'failure' - it can be ignored because we know that we need to wait a bit more. But I think that for debugging purposes it is good to at least log them as debug messages. I think keeping them as errors would be very misleading as every initialization would get quite a bit of these errors confusing users that something is wrong while this behaviour is expected.